client/src/component/User.css

```diff
@@ -0,0 +1,32 @@
+.user-container {
```

Comment:
the css is great just we need to consider some point for later ?
- how to name class ?
- we need some default style and root 
-for wireframe we wont be adding any color just that two color on the figma

Reply:
Agree with you, for later it is real need to discuss class name.

Reply:
Colors are deleted for MVP

Reply:
In App. css there is a style
 * {
    padding: 10px;
    margin: 0;
    box-sizing: border-box;
}

which influence the container positionin on the page.

we discussed before that I can not change styly for my component if it is in another components. Am I right in understanding? we need discuss it.

Reply:
Yes we can talk through naming options and css in general this weekend